packages/@misk/simpleredux/webpack.config.js

```diff
@@ -34,13 +29,9 @@ module.exports = {
         loader: "ts-loader",
         options: {
           getCustomTransformers: () => ({
-            before: [StyledComponentsTransformer]
+            before: []
```

Comment:
What does this change do?

Reply:
Is this transformer giving us errors?

Reply:
So this line removes https://www.npmjs.com/package/typescript-plugin-styled-components from the simpleredux package. I removed this since this package wasn't actually using styled-components to begin with.